### DISCUSSION TITLE
How can I focus a Textinput in React Native?

### DISCUSSION THREAD

**Discussion by apilarteq** [react-hook-form/react-hook-form#8863] [Q&A] [answered]
I've tried setFocus but I get an error, I've saw that isn't possible focus on React Native with hook form, but I want to know if there's a way, please help me guys.

**Comment by Moshyfawn** [comment, marked as answer]
You need to register your input `ref` _(pass `field.ref` to the input's `ref`)_. If it doesn't work, then we need a repro since we can't debug it with words
